_episodes/02-communicate-contributors.md

```diff
@@ -98,19 +98,20 @@ section on ["learning to say no"](https://opensource.guide/best-practices/#learn
 > ## Practice Saying No (10 minutes)
 > Choose one of the open issues linked below. Compose a 2-3 sentence response that follows the guidelines above. 
 > Add your response to the Etherpad.
-> * [lc-shell - Suggest adding Hello World](https://github.com/LibraryCarpentry/lc-shell/issues/143)
-> * [python-novice-inflammation - a couple of suggestions after reading the lesson](https://github.com/swcarpentry/python-novice-inflammation/issues/391)
-> 
+> * [Adding contents related to personal/lab server](https://github.com/datacarpentry/cloud-genomics/issues/111)
+> * OPTIONS
```

Comment:
I'll see if I can find something to add here.

Reply:
Thank you @vinisalazar. I got a lot of good advice for this section from the Curriculum Team, which I'll be incorporating in a separate PR - no need to spend more time looking for examples.